pkgs/development/python-modules/bidict/default.nix

```diff
@@ -28,31 +25,37 @@ buildPythonPackage rec {
     hash = "sha256-WE0YaRT4a/byvU2pzcByuf1DfMlOpYA9i0PPrKXsS+M=";
   };
 
-  nativeBuildInputs = [
+  build-system = [
     setuptools
-  ];
-
-  propagatedBuildInputs = [
-    sphinx
+    wheel
   ];
 
   nativeCheckInputs = [
     hypothesis
-    py
+    pytest
```

Comment:
We already provide `pytest` through the `pytestCheckHook`.

Reply:
Thanks for the quick response and for catching that! Removed `pytest` in the latest revision. Is this now good to merge?